modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1221,6 +1221,39 @@ CV__DNN_INLINE_NS_BEGIN
         CV_PROP_RW int ddepth;   //!< Depth of output blob. Choose CV_32F or CV_8U.
         CV_PROP_RW DataLayout datalayout; //!< Order of output dimensions. Choose DNN_LAYOUT_NCHW or DNN_LAYOUT_NHWC.
         CV_PROP_RW ImagePaddingMode paddingmode;   //!< Image padding mode. @see ImagePaddingMode.
+        CV_PROP_RW Scalar borderValue;   //!< Value used in padding mode for padding.
+
+        /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+         *  @param rBlob rect in blob coordinates.
+         *  @param sizeOri original input image size inserted in blob.
+         *  @param param struct of Image2BlobParams, contains all parameters needed by processing of image to blob
+         *  @returns rectangle in original image coordinates.
+         */
+        CV_WRAP Rect blobRectToImageRect(Rect rBlob, Size sizeOri);
+
+        /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+         *  @param rBlob rect in blob coordinates.
+         *  @param rImg result rect in image coordinates.
+         *  @param size spatial size for output image
+         *  @param param input image.
+         */
+        CV_WRAP void blobRectsToImageRects(std::vector<Rect> rBlob, CV_OUT std::vector<Rect>& rImg, const Size& size);
+
+        /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+         *  @param rBlob rect in blob coordinates.
+         *  @param rImg result rect in image coordinates.
+         *  @param size spatial size for output image
+         *  @param param input image.
+         */
+        CV_WRAP void blobRectsToImageRects(std::vector<Rect2d> rBlob, CV_OUT std::vector<Rect2d>& rImg, const Size& size);
+
+        /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+         *  @param rBlob rect in blob coordinates.
+         *  @param sizeOri original input image size inserted in blob.
+         *  @param param struct of Image2BlobParams, contains all parameters needed by processing of image to blob
+         *  @returns rectangle in original image coordinates.
+         */
+        CV_WRAP Rect2d blobRectToImageRect(Rect2d rBlob, Size sizeOri);
```

Comment:
Just to be sure everything is OK. Can you please add a simple binding test to https://github.com/opencv/opencv/blob/4.x/modules/dnn/misc/python/test/test_dnn.py ? Without accuracy check, just call the method and check output dtype (integer output for the integer input, double for doubles)